langchain/document_loaders/gutenberg.py

```diff
@@ -0,0 +1,25 @@
+"""Loader that loads .txt web files."""
+from typing import List
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+
+class GutenbergLoader(BaseLoader):
+    """Loader that uses urllib to load .txt web files."""
+
+    def __init__(self, file_path: str):
+        """Initialize with file path."""
+        assert file_path.startswith("https://www.gutenberg.org"), "file path must start with 'https://www.gutenberg.org'"
```

Comment:
for production code, lets use:

```
if not file_path.startswith():
   raise ValueError
```

asserts mostly good for unit tests or internal consistency checks, not validating user input